### ISSUE TITLE
small bug in OrderDetail in sendMessage form - missing confirmation 

### ISSUE THREAD

**Issue by Eva-F** [PrestaShop/PrestaShop#23195]
#### Describe the bug

in  "Order"  view.tpl is prepared the confirmation if the message should send also to a  customer

![ps37](https://user-images.githubusercontent.com/3242659/107408470-50d6d180-6b0b-11eb-9965-65ae8f2ccf61.png)

function getE returns element by given ID

![ps35](https://user-images.githubusercontent.com/3242659/107408808-bb880d00-6b0b-11eb-9d50-6785be25dd88.png)

(but "visibility" is not id attribute, it is name attribute)

therefore the condition  
````
if (getE('visibility').checked == true) 
````
invokes javascript error
![ps36](https://user-images.githubusercontent.com/3242659/107408713-9abfb780-6b0b-11eb-86dd-65c655d52423.png)

#### Expected behavior
imho should be
````
if (getE('visibility_on').checked == true) 
````

#### Steps to Reproduce

1. Go to 'Order detail'
2. Write message, switch on "Display to customer?"
3. press  send message - no confirm appears
4. See error in javascript debugger

#### Additional information

* PrestaShop version: 1.7.6.8

**Comment by hibatallahAouadni** [comment]
Hello @Eva-F 

I reproduced the issue with PS1.7.6.8 but I didn't manage to reproduce with PS1.7.7.1, see the attached screen record below:

https://drive.google.com/file/d/18mQDtPaNhnqvP4Kn5sCWe4MQxCrABBuF/view

Please check and feedback.
PS: the messages whose displayed in the console belongs to the screen castify (the screen record software that I use)

Thanks!

**Comment by Eva-F** [comment]
yes, because there is "Display to customer?" as checkbox - not as a switch (=radio)
ok - then it is solved in  1.7.7.1

**Comment by hibatallahAouadni** [comment]
Hello @Eva-F 

Thanks for your feedback!
So, I close this issue cause it's fixed with 1.7.7.0.

Thanks!
